GNS3/gns3-gui
# Project import/export/delete in cloud files

Adds the ability to:
- export the current project to cloud files
- import a project from cloud files
- delete a project that is saved in cloud files

To test:
- go to the file menu and select "Export project".  The current project will be exported to the cloud.
- Delete the project file on your local system
- Go to the file menu and select "Import project".  Select the project name and click import.  The project will be downloaded and save to your projects folder.
- Open the project as you normally would and verify that the devices you had before are still present.

One thing that might trip users up is when you export a project from the cloud, it does not open the project locally.  You still have to open the project which is now stored locally.  This can be changed if needed.
